book/security.rst

```diff
@@ -857,10 +859,10 @@ in this chapter).
 Securing other Services
 .......................
 
-In fact, anything in Symfony can be protected by doing something similar
-to this. For example, suppose you have a service (i.e. a PHP class) whose
-job is to send emails. You can restrict use of this class - no matter where
-it's being used from - to only certain users.
+Anything in Symfony can be protected by doing something similar as you used to
+secure the controller. For example, suppose you have a service (i.e. a PHP
```

Comment:
... similar to the code used to secure the controller

Your English was definitely ok, but I read it wrong at first (i read it as the *other* used - like "I used to go to the park, but not anymore" - English!)